test/acceptance/integration/notifications/mongo/sth_aggregated.feature

```diff
@@ -0,0 +1,225 @@
+  Examples:
+    | service_path            | content |
+    | /serv60100              | json    |
+    | /SERV60120              | json    |
+    | /Serv_6140              | json    |
+    | /12345678900            | json    |
+    | /12345678900            | json    |
+    | /                       | json    |
+    | with max length allowed | json    |
+    | /serv60100              | xml     |
+    | /SERV60120              | xml     |
+    | /Serv_6140              | xml     |
+    | /12345678900            | xml     |
+    | /12345678900            | xml     |
+    | /                       | xml     |
+    | with max length allowed | xml     |
+
+  @resource @ISSUE_407 @skip
+  Scenario Outline: received a notification at Cygnus and store it as aggregated values with several entities types and entities id
+    Given copy properties.json file from "epg_properties.json" to test "sth-sink" and sudo local "false"
+    And configuration of cygnus instances with different ports "true", agents files quantity "1", id "test" and in "row" mode
+    And copy flume-env.sh, matching table file from "matching_table.conf", log4j.properties, krb5.conf and restart cygnus service. This execution is only once "true"
+    And verify if cygnus is installed correctly
+    And verify if mongo is installed correctly
+    And service "test_resource", service path "/test", resource "<resource>", with attribute number "1", attribute name "random" and attribute type "celcius"
+    And receives a notification with attributes value "random number=2", metadata value "True" and content "<content>"
+    Then I receive an "OK" http code
+    And validate that the aggregated value is generate by resolution "month" in mongo
+  Examples:
+    | resource                | content |
+    | Room2_Room              | json    |
+    | Room2_HOUSE             | json    |
+    | Room2_                  | json    |
+    | ROOM_house              | json    |
+    | modelogw.assetgw_device | json    |
+    | with max length allowed | json    |
+    | Room2_Room              | xml     |
+    | Room2_HOUSE             | xml     |
+    | Room2_                  | xml     |
+    | ROOM_house              | xml     |
+    | modelogw.assetgw_device | xml     |
+    | with max length allowed | xml     |
+
+  @attribute_name
+  Scenario Outline: received a notification at Cygnus and store it as aggregated values with several attributes names
+    Given copy properties.json file from "epg_properties.json" to test "sth-sink" and sudo local "false"
+    And configuration of cygnus instances with different ports "true", agents files quantity "1", id "test" and in "row" mode
+    And copy flume-env.sh, matching table file from "matching_table.conf", log4j.properties, krb5.conf and restart cygnus service. This execution is only once "true"
+    And verify if cygnus is installed correctly
+    And verify if mongo is installed correctly
```

Comment:
This steps that are repeated in each scenario could be place in a the "Background" call like: 

```
Background:
    Given this instances are up an running
      | instances                  |
      | CYGNUS,MONGO    | 
```